charts/datadog/templates/agent-services.yaml

```diff
@@ -68,3 +68,37 @@ spec:
   - port: 443
     targetPort: 8000
 {{ end }}
+
+{{- if .Values.datadog.serviceTopology.enabled -}}
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ .Values.datadog.serviceTopology.serviceName | quote }}
+  labels:
+    app: "{{ template "datadog.fullname" . }}"
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}"
+    release: {{ .Release.Name | quote }}
+    heritage: {{ .Release.Service | quote }}
+    helm.sh/chart: "{{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}"
+    app.kubernetes.io/name: "{{ template "datadog.fullname" . }}"
+    app.kubernetes.io/instance: {{ .Release.Name | quote }}
+    app.kubernetes.io/managed-by: {{ .Release.Service | quote }}
+    app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
```

Comment:
note for later we should have an help function to this repetitive list of labels.

Reply:
good point, I'll backlog something